### PR TITLE
Associations

### PR DESCRIPTION
**Pull request by OscarFMdev**
# PR Description 📄 

In this PR I:
- Created classes Classroom, Book and Rental 🏫 
- Added associations to these classes. ✅ 

### Thank you for reviewing my code! 🙌🏽

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -15,3 +18,9 @@ def play_hooky
 puts student1.age # 10
 # puts student1.classroom # A
 puts student1.play_hooky # ¯\(ツ)/¯
+
+puts student1.classroom
+
+classroom1 = Classroom.new('A')
+
+puts classroom1.add_student(student1)
```
Comment (mugishasam123) [inline student.rb:31]: - [x] Please try to group operations in a method to make your code look clean .like you can create a method called `classroom` and you perform those operations:

```
 def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end

```

**Review by mugishasam123 — CHANGES_REQUESTED** [review]
Hi @OscarFMdev  👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
alt
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors 👍 
-  Descriptive Pr, keep rocking 🚀 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: student.rb (hunk 1 above)
Comment (OscarFMdev) [reply student.rb:31]: @mugishasam123 thank you for letting me know!

Comment (OscarFMdev) [reply student.rb:31]: @mugishasam123 thank you for letting me know!

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @OscarFMdev,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by OscarFMdev** [comment]
Thanks @Omar-Muhamad !

**Pull request merged**